docs(ET-1813): add object storage configuration of reader (#6470)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx

# but the env structure looks like this
#export DBZ_DATABASES_1__PASSWORD=password

##############################################################################

> # DMS Reader Object storage config, only necessary for appliance       #

Author: Thanks, the object should be lowercase. And using HCP is proper

The Apicurio client uses it to perform mTLS authentication with the `transporter-rw-service`.

### `AWS_ACCESS_KEY_ID`

> 1) If you are using AWS S3, You need to set the access key id with the read and write permission of the bucket.

Reviewer: What do you think about the following suggestion? 
```suggestion
1) If you are using AWS S3, set this value to an access key ID that has read and write permissions for the S3 bucket.
```

> 2) If you are using s3 compatible storage like minio, you need to set the access key of your object storage.

Reviewer: ```suggestion
2) If you are using S3-compatible storage, like MinIO, set this value to the access key for your object storage.
```

### `AWS_SECRET_ACCESS_KEY`

> 1) If you are using AWS S3, You need to set the secret access key with the read and write permission of the bucket.

Reviewer: ```suggestion
1) If you are using AWS S3, set this value to a secret access key that has read and write permissions for the specified S3 bucket.
```

> 2) If you are using s3 compatible storage like minio, you need to set the secret key of your object storage.

Reviewer: ```suggestion
2) If you are using S3-compatible storage, like MinIO, set this value to the secret key for your object storage.
```

> ### `AWS_REGION (Optional)`

Reviewer: ```suggestion
### `AWS_REGION` (Optional)
```

> 1) If you are using AWS S3, You need to set this according to the region in AWS_ENDPOINT_URL_S3

Reviewer: ```suggestion
1) If you are using AWS S3, set this value to match the region specified in AWS_ENDPOINT_URL_S3.
```

> 2) If you are using s3 compatible storage like minio, you don't need to set this.

Reviewer: ```suggestion
2) If you are using S3-compatible storage, such as MinIO, this setting is not required.
```

> ### `AWS_CA_BUNDLE (Optional)`

Reviewer: ```suggestion
### `AWS_CA_BUNDLE` (Optional)
```

> This is only needed if you are using a self-signed certificate for the object storage. You need to set the path to the CA bundle file.

Reviewer: ```suggestion
Set this value only if you are using a self-signed certificate for your object storage. Specify the path to the CA bundle file.
```

> You don't need to set this if you are using AWS S3 or s3 compatible storage with publicly-trusted CAs.

Reviewer: ```suggestion
This setting is not required if you are using AWS S3 or S3-compatible storage with publicly trusted certificate authorities (CAs).
```

### `AWS_ENDPOINT_URL_S3`

> 1) If you are using AWS S3, you can find the endpoint url in: https://docs.aws.amazon.com/general/latest/gr/s3.html

Reviewer: ```suggestion
1) If you are using AWS S3, you can find the endpoint URL in the AWS documentation: [AWS S3 Endpoints](https://docs.aws.amazon.com/general/latest/gr/s3.html).
```

> 2) If you are using s3 compatible storage like minio, you need to set according to your object storage configuration.

Reviewer: ```suggestion
2) If you are using S3-compatible storage, like MinIO, set this value according to your object storage configuration.
```

### `BUCKET_NAME`

> The bucket name you want to use for the migration. The bucket need to be created before the migration.

Reviewer: ```suggestion
Specify the name of the bucket you want to use for the migration. The bucket must be created before starting the migration.
```
For me, bucket is specific to AWS. Should we say that if the have S3-compatible storage, like MinIO, this values doesn't need to be filled? In that case: 
```suggestion
Specify the name of the bucket you want to use for the migration. The bucket must be created before starting the migration. This setting is only required for AWS instances.
```

Author: Bucket is a common concept, so both aws s3 and s3-compatible object storage need a Bucket created before the migration startes. So I will chose the first suggestion
